data/hails.txt

```diff
@@ -2479,6 +2479,38 @@ phrase "hostile bounty"
 		"off beam."
 		"mislead."
 
+
+phrase "friendly quarg"
+	word
+		"We"
+		"The Quarg"
+		"Our people"
+	word
+		" "
+	word
+		"guide and guard"
+		"watch"
+		"oversee"
+		"look after"
+		"safeguard"
+	word
+		" "
+	word
+		"the galaxy"
+		"the young and foolish"
+		"the young races"
+		"the galaxy's children"
+		"those who live"
+	word
+		" "
+	word
+		"as we are sworn to do."
+		"as the Drak guide us."
+		"to uphold the sanctity of life."
+		"as is our geis."
```

Comment:
It is an incredibly obscure word.
I couldn't find any internet dictionary that contained it.
I could only find it on Wikipedia and Wiktionary.
I do believe we should use a different word.

At least stuff like "sanctity" can be figured out from their roots, with some effort; "geis" is basically impossible to decipher.